### PR TITLE
Updated the getting started docs [CN-234]

### PR DESCRIPTION
**Pull request by alparslanavci**
- Removed `bundle.yaml` content, instead provided the direct link to the raw file in the command.
- Updated with the new naming
- Added Hazelcast Platform OSS installation info
- Updated the status info by providing commands
- Moved the `Connecting outside Kubernetes` section under `Deploy & Manage`

### CODE REVIEW

File: docs/modules/ROOT/pages/get-started.adoc [hunk 1]
```diff
@@ -101,75 +77,69 @@ Members {size:3, ver:3} [
 ]
 ```
 
-== Connect to Hazelcast from Outside Kubernetes
+== Start Hazelcast Enterprise Cluster
 
-To connect to a Hazelcast cluster from outside Kubernetes, you need to configure the `expose-externally` field of your custom resource.
+Hazelcast Enterprise requires having the license key specified as Kubernetes Secret.
 
+NOTE: If you don't have Hazelcast Enterprise License Key, please check xref:deploy:using-enterprise-edition.adoc#requesting-a-license-key[Requesting a License Key]
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/get-started.adoc:84]:
```suggestion
NOTE: If you don't have Hazelcast Enterprise License Key, please check xref:hazelcast:deploy:using-enterprise-edition.adoc#requesting-a-license-key[Requesting a License Key]
```

Comment (JakeSCahill) [reply docs/modules/ROOT/pages/get-started.adoc:84]: Now that these docs are in their own subsite (Antora component), we need to reference the `hazelcast` component to link to the Platform docs.

Comment (JakeSCahill) [reply docs/modules/ROOT/pages/get-started.adoc:84]: I updated the link to go straight to the Hazelcast website. The docs are just an extra step that users don't need.

File: docs/modules/ROOT/pages/get-started.adoc [hunk 2]
```diff
@@ -59,20 +46,9 @@ kubectl apply -k .
 
 Note that you'll need to add a parameter `-n <your-namespace>` to all further commands.
 
-== Create Secret with Hazelcast License Key
-
-Hazelcast Enteprise requires having the license key specified as Kubernetes Secret.
+== Start Hazelcast cluster
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/get-started.adoc:49]: We use title case for headings.

```suggestion
== Start Hazelcast Cluster
```

File: docs/modules/ROOT/examples/hazelcast-enterprise.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: hazelcast-sample
+spec:
+  clusterSize: 3
+  repository: 'docker.io/hazelcast/hazelcast-enterprise'
+  version: '{full-version}-slim'
```
Comment (JakeSCahill) [inline docs/modules/ROOT/examples/hazelcast-enterprise.yaml:8]: `full-version` contains the `major.minor.patch` version of the operator.

Is this what we want to use?

File: bundle.yaml [hunk 4]
```diff
@@ -78,7 +78,7 @@ spec:
                 description: Repository to pull the Hazelcast Platform image from.
                 type: string
               version:
-                default: '{page-latest-supported-hazelcast}-slim'
+                default: "5.0"
```
Comment (JakeSCahill) [inline bundle.yaml:81]: Something to consider:

Moving the `bundle.yaml` to the root means we can't use attributes to keep the Platform version in sync with the rest of the docs. We would need to remember to update this field separately for each new release.

To be able to use these attributes, we could host the `bundle.yaml` in the `modules/ROOT/attachments` directory. This way the public URL to get the file will be `docs.hazelcast.com/hazelcast-operator/{version}/_attachments/bundle.yaml`

**Review by JakeSCahill — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/examples/hazelcast-enterprise.yaml (hunk 3 above)
Comment (alparslanavci) [reply docs/modules/ROOT/examples/hazelcast-enterprise.yaml:8]: You are right, we don't want to use operator version here. Can we add a compatible Hazelcast version parameter, so we can use it where it is needed?

File: bundle.yaml (hunk 4 above)
Comment (alparslanavci) [reply bundle.yaml:81]: Again, you are right. However, we are planning to update this `bundle.yaml` on each release of the docs with the newly generated one. Thus, we can keep it as it is for now.

**Review by hasancelik — APPROVED** [review]

**Review by SeriyBg — APPROVED** [review]

File: docs/modules/ROOT/examples/hazelcast-enterprise.yaml (hunk 3 above)
Comment (JakeSCahill) [reply docs/modules/ROOT/examples/hazelcast-enterprise.yaml:8]:
```suggestion
  version: '{page-latest-supported-hazelcast}-slim'
```

**Review by JakeSCahill — APPROVED** [review]

**Pull request merged**